h/static/scripts/tests/util/submit-form-test.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+var submitForm = require('../../util/submit-form');
+
+/**
+ * Fake `fetch()` implementation.
+ *
+ * This does not mock the global `window.fetch` function but provides a mock
+ * via `this.fetch` which can be passed as a dependency to functions that make
+ * network requests.
+ */
+class FakeFetch {
```

Comment:
Ok, I didn't realise that Chrome `fetch` wasn't a wrapper around XMLHttpRequest. In that case let's not implement something [that already exists](https://www.npmjs.com/package/fetch-mock) for ourselves.